cobc/ppparse.y

```diff
@@ -72,15 +73,36 @@ static enum cb_directive_action		current_cmd = PLEX_ACT_IF;
 /* Local functions */
 
 static char *
-fix_filename (char *name)
+unquote_alnum (char *name)
 {
 	/* remove quotation from alphanumeric literals */
 	if (name[0] == '\'' || name[0] == '\"') {
```

Comment:
it would be reasonable to just check `name[0] == name [size - 1]` and only do the unquote in this case, otherwise leave as-is